RequestResponseModels/Ticket/Response/TicketResponse.cs

```diff
@@ -40,7 +40,7 @@ public TicketResponse(int id, string title, string body, string type, string pri
 
         public DateTime? UpdatedAt { get; set; }
 
-        public byte[]? Attachements { get; set; }
+        public byte? Attachements { get; set; }
```

Comment:
An attachment can't be just a byte...

Reply:
I know, just modified so it works. That is not my code :D